Rawl_B502/de_genelogia_dáil_coirpri_arad_retro.ttl

```diff
@@ -0,0 +1,205 @@
+    irishRel:genName "Máil";
+    rel:childOf <#Ainmerech>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Maeil>.
+
+<#Ainmerech>
+    a foaf:Person;
+    irishRel:nomName "Ainmere";
+    irishRel:genName "Ainmerech";
+    rel:childOf <#FirRoith>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Ainmerech>.
+
+<#FirRoith>
+    a foaf:Person;
+    irishRel:nomName "Fer Roth";
+    irishRel:genName "Fir Roith";
+    rel:childOf <#Muine>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#FirRoith>.
+
+<#Muine>
+    a foaf:Person;
+    irishRel:nomName "Muine";
+    irishRel:genName "Muine";
+    rel:childOf <#FirNeud>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Muine>.
+
+<#FirNeud>
+    a foaf:Person;
+    irishRel:nomName "Fer Neud";
+    irishRel:genName "Fir Neud";
+    rel:childOf <#FirLugdach>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#FirNued>.
+
+<#FirLugdach>
+    a foaf:Person;
+    irishRel:nomName "Fer Lugdach";
+    irishRel:genName "Fir Lugdach";
+    rel:childOf <#Buain>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#FirLugdach>.
+
+<#Buain>
+    a foaf:Person;
+    irishRel:nomName "Buan";
+    irishRel:genName "Buain";
+    rel:childOf <#Argatibair>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Buain>.
+
+<#Argatibair>
+    a foaf:Person;
+    irishRel:nomName "Argatibar";
+    irishRel:genName "Argatibair";
+    rel:childOf <#CairpreCluichechair>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Argatibair>.
+
+<#CairpreCluichechair>
+    a foaf:Person;
+    irishRel:nomName "Cairpre Cluichechair";
+    irishRel:genName "Cairpre Cluichechair";
+    rel:childOf <#ConCorb>;
+    owl:sameAs <http://example.com/Rawl_B502/de_genelogia_dáil_coirpri_arad.ttl#CairpreCluichechair>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#CorpriCluchechair>.
+
+<#ConCorb>
+    a foaf:Person;
+    irishRel:nomName "Cú Corb";
+    irishRel:genName "Con Corb";
+    rel:antagonistOf <#Arttn-Óenfer>;
+    owl:sameAs <http://example.com/Rawl_B502/de_genelogia_dáil_coirpri_arad.ttl#ConCorb>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#ConCorbb>.
+
+<#Arttn-Óenfer>
+    a foaf:Person;
+    irishRel:accName "Artt n-Óenfer"
+    irishRel:nomName "Art Óenfer";
+    owl:sameAs <http://example.com/LL/rig_ailig.ttl#ArtOenfhir>.
+
+<#Laider>
+    a foaf:Person;
+    irishRel:nomName "Laider";
+    foaf:title "ara"@sga, "charioteer"@eng;
+    rel:workedFor <#ConCorb>;
+    owl:sameAs <http://example.com/Rawl_B502/de_genelogia_dáil_coirpri_arad.ttl#Laider>.
+
+<#Ethni>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:nomName "Ethni";
+    owl:sameAs <http://example.com/Rawl_B502/de_genelogia_dáil_coirpri_arad.ttl#Eithne>.
+
+<#Oengus>
+    a foaf:Person;
+    irishRel:nomName "Oengus";
+    owl:sameAs <http://example.com/Rawl_B502/de_genelogia_dáil_coirpri_arad.ttl#ÓengusaMúisc>;
+    rel:relationship [
```

Comment:
In this instance, Óengus is giving land to his son-in-law, Cairpre. We discussed creating an RDF relationship for this kind of situation. I'll use "foaf:knows" for now but will tag it for future attention.